index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="scripts/main.js" defer></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/mainStyle.css">
+    <title>Stephen's portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav class="toolbar">
+            <ul>
+                <li class="textPlaceholder"> <a href="#" aria-label="Ssenkooto Stephen"> Ssenkooto Stephen </a> </li>
+                <li> <button id="menuButton" type="button"> <img src="icons/menu.svg" alt=""> </button></li>
+                <li id="navigation">
+                    <ul class="container">
+                        <li class="textPlaceholder"><a href="#projects" aria-label="Portfolio"> Portfolio</a></li>
+                        <li class="textPlaceholder"><a href="#about" aria-label="About"> About</a></li>
+                        <li class="textPlaceholder"><a href="#contact" aria-label="Contact"> Contact</a></li>
+                        <li class="textPlaceholder"><a rel="noopener noreferrer" href="mailto:ssenkootos74@gmail.com"
+                                aria-label="Send Email"><img src="icons/mail.svg" alt="Send Email"></a></li>
+                    </ul>
+                </li>
+            </ul>
+        </nav>
+    </header>
```

Comment:
- [ ] Your navbar in the mobile version has the desktop links which should not be the case. It would be great If you remove the desktop links from the mobile version and position the logo and hamburger as in the template 💯
![f1](https://user-images.githubusercontent.com/85113567/150105819-051a6191-9066-4b29-96a8-10345f17d815.jpg)